.github/workflows/lint-test.yml

```diff
@@ -12,6 +15,7 @@ env:
 
 jobs:
   lint:
+    name: lint
```

Comment:
Why would you need this when we have the job defined on the line above this?  It doesn't look like you had to add it for test.